src/server/svr_func.c

```diff
@@ -5073,7 +5100,7 @@ offline_all_provisioning_vnodes()
  *
  */
 
-static void
+static int
```

Comment:
Please update doxygen, also we can return a different rc value from this function instead of -5. Like 0 for success and -1 for failure.